themes/socialbase/components/02-atoms/alert/alert.scss

```diff
@@ -56,8 +56,8 @@
   // Adjust close link position
   .close {
     position: relative;
```

Comment:
Lets remove the `position: relative` as well so it remains in the normal page flow. With the margin changes we don't need it anymore :)